pkg/dispatcher/config.go

```diff
@@ -196,6 +198,15 @@ func isSSHBastionJob(base prowconfig.JobBase) bool {
 	return false
 }
 
+// DetermineCloudMapping determines if for a given cloud there is a replacement to map, eg for cost saving reasons
+func (config *Config) DetermineCloudMapping(jobBase prowconfig.JobBase) string {
+	cloud := DetermineCloud(jobBase)
+	if mapping, ok := config.CloudMapping[api.Cloud(cloud)]; ok {
+		cloud = string(mapping)
+	}
+	return cloud
```

Comment:
My bad. I was trying to avoid `cloud := DetermineCloud(jobBase)` but found it is used in the `if`.